gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/public/src/components/privacySettings/js/PrivacySettings.js

```diff
@@ -213,22 +288,48 @@ var gpii = gpii || {};
         dialogForRemoval.open();
     };
 
-    gpii.oauth2.privacySettingsWithPrefs.removeDecision = function (dialog, url, type, authDecisionId) {
+    gpii.oauth2.privacySettings.removeDecision = function (dialog, url, type, authDecisionId, removalSuccessEvt) {
```

Comment:
This should at least be factored to use the gpii.oauth2.ajax utility for consistenty, so we can replace all usages at the same time.